src/test/java/io/strimzi/kafka/bridge/http/HttpBridgeTest.java

```diff
@@ -2256,10 +2270,56 @@ public void seekToBeginningAndReceive(TestContext context) {
 
         seekAsync.await();
 
+        // Consumer instance not found
+        Async instanceNotFoundAsync = context.async();
+        String uriWithNotExistIstance = "http://" + BRIDGE_HOST + ":" + BRIDGE_PORT + "/consumers/" + groupId + "/instances/" + "not-exist-instance";
```

Comment:
why are testing this here and not with another test method?

Reply:
I did it because of the same pre-conditions for test cases, but.. if it confuses someone I will move it to separate test method

Reply:
done